zboxcore/sdk/allocation.go

```diff
@@ -638,12 +652,6 @@ func (a *Allocation) ListDirFromAuthTicket(authTicket string, lookupHash string)
 }
 
 func (a *Allocation) ListDir(path string) (*ListResult, error) {
-	consensusThresh := (float32(a.DataShards) * 100) / float32(a.DataShards+a.ParityShards)
-	fullconsensus := float32(a.DataShards + a.ParityShards)
-	return a.listDir(path, consensusThresh, fullconsensus)
-}
-
-func (a *Allocation) listDir(path string, consensusThresh, fullconsensus float32) (*ListResult, error) {
```

Comment:
IMHO do not remove this private function completely, keep this as ListDir() calling listDir()

Reply:
If you see then it makes no difference. 
The unexported listDir function just required consensuses but now it doesn't require to be passed as function parameter.
It is passed from allocation object

Reply:
It's very common in Go to return a private implementation of a public interface, even if there are no changes in params. That pattern is used throughout the stdlib (or even throughout gosdk).